packages/credentials/src/holder.ts

```diff
@@ -55,10 +55,10 @@ function pointerToAttributeName(
  * @param credential A Verifiable Credential containing a proof.
  * @param proofOptions Options for creating the derived proof.
  * @param proofOptions.disclose Allows selecting which claims/attributes in the credential should be disclosed, hiding others.
- * @param proofOptions.disclose.only An array of credentialPath expressions selecting attributes to be disclosed.
+ * @param proofOptions.disclose.only An array of JSON-path (IETF RFC 6901) expressions selecting attributes to be disclosed.
```

Comment:
Could we link to the spec here?

Reply:
good idea, also bc it made me realise that it's actually JSON Path that we're using, not JSON pointers 😬

Reply:
4f389db22224240d0bf025a7fd03243709d2fcc0